geomstats/geometry/discrete_surfaces.py

```diff
@@ -67,6 +73,10 @@ def __init__(
             equip=equip,
         )
 
+    def new(self, equip=True):
```

Comment:
Why having `new` in DiscreteSurfaces but not in any other Manifold?

Reply:
All the spaces that are used as total space in the context we do not have explicit representation of base need to define this method (see e.g. `DiscreteCurvesStartingAtOrigin`).